src/restoreImpl.ts

```diff
@@ -69,7 +70,11 @@ async function restoreImpl(
         );
 
         core.setOutput(Outputs.CacheHit, isExactKeyMatch.toString());
-        core.info(`Cache restored from key: ${cacheKey}`);
+        if (lookupOnly) {
+            core.info(`Cache would have been restored from key: ${cacheKey}`);
```

Comment:
```suggestion
            core.info(`Cache found and can be restored from key: ${cacheKey}`);
```

To keep the logs uniform?

<img width="1400" alt="Screenshot 2023-03-09 at 3 56 46 PM" src="https://user-images.githubusercontent.com/98868223/223996456-f5251552-09f2-4b8e-8fe3-a69f51565ac0.png">